src/fillTank.test.js

```diff
@@ -1,11 +1,87 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
```

Comment:
1. Unify naming approach: start the name everywhere with uppercase or lowercase. 
2. Mention in the summary that the fuelRemains and money should be updated